.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Hi there! If you are here to report a bug, or to discuss a feature (new or existing), you can use the below template to get started quickly. Fill out all those parts which you're comfortable with, and delete the remaining ones.
+-->
+
+#### Nature of issue?
+
+<!-- Select any one issue and delete the other two -->
+
+- Found a bug
+- Existing feature enhancement
+- New feature request
+
+<!-- If you found a bug, the following information might prove to be helpful for us. Simply remove whatever you can't determine/don't know. -->
+#### Details about the bug: 
+
+- Web browser and version: <!-- On Chrome/FireFox/Opera you can enter "about:" in the address bar to find out the version -->
+- Operating System: <!-- Ex: Windows/MacOSX/Linux along with version -->
+- Steps to reproduce this bug:
+
+<!-- Include a simple code snippet that demonstrates the problem, along with any console errors produced. If this isn't possible, then simply describe the issue as best you can! Feel free to link to the web editor or include pictures or a video. -->
+
+<!-- If you want to enhance an existing feature, please describe here, otherwise remove this section -->
+#### Feature enhancement details:
+
+
+<!-- If you want to request a new feature, please describe here, otherwise remove this section -->
+#### New feature details:
```

Comment:
Instead, we can use something like https://github.com/vega/vega-lite/tree/master/.github/ISSUE_TEMPLATE. 

My main problem with this PR is that nothing in this description is specific to the vega editor so I don't find it useful right now.

Reply:
Hm! Sure , Should I change the template or leave this for future?

Reply:
I have not yet had any problems with how people reported issues, to be honest. 

Does this solve a particular problem you experienced? If not, let's close it.

Reply:
No , I guess we can just close this one now